toonz/sources/tnztools/brushtool.cpp

```diff
@@ -2616,12 +2595,6 @@ void BrushData::loadData(TIStream &is) {
       is >> m_join, is.matchEndTag();
     else if (tagName == "Miter")
       is >> m_miter, is.matchEndTag();
-    else if (tagName == "Frame_Range")
-      is >> m_frameRange, is.matchEndTag();
-    else if (tagName == "Snap")
-      is >> m_snap, is.matchEndTag();
```

Comment:
Looking how another boolean values are loaded, maybe this line should be 
`is >> val, m_snap= val, is.matchEndTag();` ?

Reply:
@shun-iwasawa You are right.  That does fix it.  Do you think that snap and frame range should be part of the preset or left as separate values?

Reply:
Well, excluding them would be somewhat reasonable as you initially mentioned.